bool e_stop
uint8 gear
uint8 speed
int32 steer
uint8 brake